### PR TITLE
Script to Connect DB Instance With application

### PR DESCRIPTION
**Pull request by mrrajan**
This PR includes script to Connect DB Instance With application 
- Application YAML file to import
- Openshift Library to create service binding

Task: DBAAS-513

Signed-off-by: Rajan Ravi <rravi@redhat.com>

### CODE REVIEW

**Comment by mrrajan** [comment]
[test-output.zip](https://github.com/RHODA-lab/rhoda-ci/files/8833319/test-output.zip)
Dry run results

**Comment by mrrajan** [comment]
[test-output.zip](https://github.com/RHODA-lab/rhoda-ci/files/8844368/test-output.zip)
Updated dry run results.

File: resources/keywords/deploy_application.resource [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+Load Data To ${database} Database From Application
+    Run Keyword If    "Mongo" in """${database}"""
+    ...    Load Data To Mongo Application
+
+Load Data To Mongo Application
+    ${appStatus}=    Run Keyword And Return Status    Element Should Be Visible    ${hdrFruitApp}
+    Run Keyword If    not ${appStatus}
+    ...    Reload Page
+    Run Keyword And Return Status    Element Should Be Visible    ${hdrFruitApp}
```
Comment (harshkumarRH) [inline resources/keywords/deploy_application.resource]: @mrrajan Could you please change this to `Wait Until Element Is Visible    ${txtBxFruitName}`

**Review by harshkumarRH — APPROVED** [review]
Looks good to me

**Review by psathyan — APPROVED** [review]

**Pull request merged**